server/server.go

```diff
@@ -40,6 +40,8 @@ type Config struct {
 	HTTPListenPort   int
 	GRPCListenPort   int
 
+	DontRegisterInstrumentation bool
```

Comment:
Ah, gotcha.

I'm leaning toward flag, because one person's internet-facing service is another person's backend service. I don't feel strongly about it though. LMK if you'd rather not, and I'll hit the button.